src/renderer/webgl.js

```diff
@@ -1011,50 +952,14 @@
 
     },
 
-    getTriangles: function(rect, triangles) {
-
-      var top = rect.top,
-          left = rect.left,
-          right = rect.right,
-          bottom = rect.bottom;
-
-      // First Triangle
-
-      triangles[0] = left;
-      triangles[1] = top;
-
-      triangles[2] = right;
-      triangles[3] = top;
-
-      triangles[4] = left;
-      triangles[5] = bottom;
-
-      // Second Triangle
-
-      triangles[6] = left;
-      triangles[7] = bottom;
-
-      triangles[8] = right;
-      triangles[9] = top;
-
-      triangles[10] = right;
-      triangles[11] = bottom;
-
-    },
-
     updateTexture: function(gl, elem) {
 
       this[elem._renderer.type].updateCanvas.call(webgl, elem);
 
-      if (elem._renderer.texture) {
-        gl.deleteTexture(elem._renderer.texture);
```

Comment:
I always wanted to know if you had to delete or if the garbage collector would handle that for you 🤔

Reply:
The GC won't handle texture deletion. Here it doesn't matter because we were always creating a new texture afterwards, however.

Reply:
Then do we need to delete it somewhere else?

Reply:
It looks like that's done [here](https://github.com/jonobr1/two.js/blob/dev/src/renderer/webgl.js#L41).

Reply:
That only happens when you remove a child from the scene. You can update the texture without removing the child from the screen. For instance if you change the verts on the path the texture will update but the `removeChild` function will not be called.

Reply:
`texImage2D` doesn't create any new textures; only `createTexture` does that. If you update an existing texture with `texImage2D`, as is done here, there's no memory leak.

Reply:
Ah, sorry I see now. Below on https://github.com/jonobr1/two.js/pull/422/files/1845721f51896e8c6196d425d398c98b7c916e97#diff-135ef49c47b024db0620d873c2fafeb5L1057 we just re-use the texture instead of creating new ones each time. Thanks so much for taking the time to help me better understand!